contracts/voting/dao-voting-cw4/src/contract.rs

```diff
@@ -164,12 +245,48 @@ pub fn query_total_power_at_height(deps: Deps, env: Env, height: Option<u64>) ->
 }
 
 pub fn query_info(deps: Deps) -> StdResult<Binary> {
-    let info = cw2::get_contract_version(deps.storage)?;
+    let info = get_contract_version(deps.storage)?;
     to_json_binary(&dao_interface::voting::InfoResponse { info })
 }
 
+pub fn query_is_active(deps: Deps) -> StdResult<Binary> {
+    let active_threshold = ACTIVE_THRESHOLD.may_load(deps.storage)?;
+
+    match active_threshold {
+        Some(ActiveThreshold::AbsoluteCount { count }) => {
+            let group_contract = GROUP_CONTRACT.load(deps.storage)?;
+            let total_weight: TotalWeightResponse = deps.querier.query_wasm_smart(
+                &group_contract,
+                &cw4_group::msg::QueryMsg::TotalWeight { at_height: None },
+            )?;
+            to_json_binary(&IsActiveResponse {
+                active: total_weight.weight >= count.into(),
+            })
+        }
+        Some(ActiveThreshold::Percentage { percent, .. }) => {
```

Comment:
Should have added a note on this, but `ActiveThreshold::Percentage` is not really needed in the case of a membership DAO. Unlike NFTs or Tokens, there is no notion of a `total_supply` or staking.

This is good because we can simplify the contract!

On instantiate, we should only allow for `ActiveThreshold` to be defined in terms of absolute count, and throw an error if someone submits a percentage.